src/components/scenes/Loans/LoanCreateConfirmationScene.js

```diff
@@ -120,7 +121,9 @@ export const LoanCreateConfirmationScene = (props: Props) => {
 
   if (loanAccountError != null) return <Alert title={s.strings.error_unexpected_title} type="error" message={translateError(loanAccountError)} />
 
-  return (
+  return loanAccount == null ? (
+    <FillLoader />
+  ) : (
```

Comment:
This may not belong in `fixup! Update AAVE scene route props to use LoanManager, update routing`. Either it applies to another commit, or stands on its own?